README.md

```diff
@@ -55,7 +55,7 @@ We suggest you use a secret to hide to app-code:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
+    projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
Do you think we need back-compat here?

Reply:
We do not, this API isn't changing. It's just the example that I changed.